exercises/practice/pythagorean-triplet/.approaches/introduction.md

```diff
@@ -1,15 +1,15 @@
 # Introduction
 
-## Hints and tips
+## General guidance
 
 - A O(n^3) solution is trivial to find, but too slow for the tests. A O(n^2) solution is pretty easy to find. A O(n) solution requires solving some simultaneous equations.
 
-## Example solutions
+## Approaches
 
 There are three reasonably common solutions to this problem, a cubic time solution, a quadratic time solution and a linear time solution
 (if those terms are unclear to you, you might like to read about [time complexity](https://en.wikipedia.org/wiki/Time_complexity)).
```

Comment:
```suggestion
There are three reasonably common solutions to this problem, a cubic time solution, a quadratic time solution and a linear time solution
(if those terms are unclear to you, you might like to read about [time complexity](https://en.wikipedia.org/wiki/Time_complexity)).
A cubic time solution is easy to find, but too slow for the tests or for many practical uses.
A quadratic time solution is pretty easy to find and will be good enough for the tests.
Finding a linear time solution will require some mathematics. The solution below was derived by solving some simultaneous equations.
```

The three suggestions above combine the "General guidance" and "Approaches" sections.